testdata/p4_16_samples_outputs/array_field1-midend.p4

```diff
@@ -6,18 +6,29 @@ extern bit<1> f(inout bit<1> x, in bit<1> b);
 control c(out H[2] h);
 package top(c _c);
 control my(out H[2] s) {
+    bit<32> a_0;
```

Comment:
These changes suggest that some analysis has become less precise.

Reply:
The code here changed because we're now assuming that the unknown method call `f` might depend on the locals.  However, f is a global extern function, so we should probably assume it only depends on/modifies things passed to it explicitly.